packages/react-components/react-avatar/src/stories/AvatarGroup/index.stories.tsx

```diff
@@ -18,7 +15,7 @@ export default {
   parameters: {
     docs: {
       description: {
-        component: [descriptionMd, bestPracticesMd].join('\n'),
+        component: [descriptionMd].join('\n'),
```

Comment:
We'll still want to include best practices eventually, right?

Reply:
yeah, I'm removing it for now since Tudor asked if we could remove it until filled out.